pycbc/inference/models/gaussian_noise.py

```diff
@@ -214,7 +215,8 @@ class GaussianNoise(BaseDataModel):
     name = 'gaussian_noise'
 
     def __init__(self, variable_params, data, waveform_generator,
-                 f_lower, psds=None, f_upper=None, norm=None,
+                 lower_frequency_cutoff=None, psds=None, 
+                 f_upper=None, norm=None,
```

Comment:
Don't make `low_frequency_cutoff` a keyword argument, as you did here. Just rename `f_lower` -> `low_frequency_cutoff`. Then you don't need the block of code below to do all of the checking.